src/github/pullRequestManager.ts

```diff
@@ -143,6 +145,11 @@ export class PullRequestManager implements vscode.Disposable {
 			}
 		}));
 
+		this._subs.push(this._git.onDidChangeState(async _ => {
+			await this.updateRepositories();
+			vscode.commands.executeCommand('pr.refreshList');

```

Comment:
Is this `pr.refreshList` redundant? If the state changes, `this.updateRepositories` will update `PullRequestManager.state` and trigger an udpate of the prs tree.